connector_prestashop/unit/mapper.py

```diff
@@ -697,31 +638,46 @@ def backend_id(self, record):
 
     @mapping
     def name(self, record):
-        binder = self.get_connector_unit_for_model(Binder, 'prestashop.supplier')
-        partner_id = binder.to_openerp(record['id_supplier'], unwrap=True)
+        binder = self.unit_for(Binder, 'prestashop.supplier')
+        partner_id = binder.to_odoo(record['id_supplier'], unwrap=True)
         return {'name': partner_id}
 
     @mapping
     def product_id(self, record):
         if record['id_product_attribute'] != '0':
-            binder = self.get_connector_unit_for_model(Binder, 'prestashop.product.combination')
-            return {'product_id': binder.to_openerp(record['id_product_attribute'], unwrap=True)}
-        binder = self.get_connector_unit_for_model(Binder, 'prestashop.product.product')
-        return {'product_id': binder.to_openerp(record['id_product'], unwrap=True)}
+            binder = self.unit_for(Binder, 'prestashop.product.combination')
+            return {'product_id': binder.to_odoo(
+                record['id_product_attribute'], unwrap=True)}
+        binder = self.unit_for(Binder, 'prestashop.product.product')
```

Comment:
`prestashop.product.product` should be another model
`self.unit_for(Binder, ...)` can be replaced by `self.binder_for(...)`